content/spin/kv-store.md

```diff
@@ -339,6 +391,55 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
 
 {{ blockEnd }}
 
+{{ startTab "Python"}}
+
+<!-- @nocpy -->
+
+```typescript
+import { HandleRequest, HttpRequest, HttpResponse } from "@fermyon/spin-sdk"
```

Comment:
And this looks suspiciously JS-like.  Is it possible the JS and Python samples have got mixed up here, maybe a copy-paste hiccup?

Reply:
This is what a new Python app looks like:

```python
ivan@hecate:~/testing/hello$ cat app.py
from spin_http import Response


def handle_request(request):

    return Response(200,
                    [("content-type", "text/plain")],
                    bytes(f"Hello from the Python SDK", "utf-8"))
```